src/test/cypressjs/cypress/integration/testRedirectURLs.js

```diff
@@ -20,7 +20,7 @@ describe('Test Redirect URLs', () => {
         // Do not know how to handle urls that are not /docs
         return;
       } else {
-        cy.visit(Cypress.env('website_url') + tokens[0])
+        cy.visit(target_url + tokens[0])
       }
```

Comment:
Can I sneak this change in so that we can also test non-html redirects:
```
      } else {
        if (tokens[0].endsWith('.html')){
          cy.visit(target_url + tokens[0])
        } else {
          cy.request(target_url + tokens[0])
        }
      }
```

Reply:
ok

Reply:
Can you commit it to the from branch?